packages-exp/app-exp/internal/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "@firebase/app-exp/internal",
+    "main": "../dist/cjs/internal.js",
+    "browser": "../dist/esm5/internal.js",
+    "module": "../dist/esm5/internal.js",
+    "esm2017": "../dist/esm2017/internal.js",
+    "typings": "../dist/esm5/packages-exp/app-exp/src/internal.d.ts"
```

Comment:
Is this path correct?

Reply:
Tested It is correct, but may move it in the future.